routes/index.js

```diff
@@ -23,6 +24,7 @@ router.post('/answer', auth_token, postAnswer.createAnswer);
 router.get('/answer/:question_id', auth_token, viewAnswer.viewAnswer);
 router.put('/answer/:answer_id', auth_token, acceptAnswer.acceptAnswer);
 router.post('/comment', auth_token, commentAnswer.commentAnswer);
+router.get('/question/:user_id', auth_token, getQuestion.getQuestion);
```

Comment:
Don't pass the `user_id` as a URL parameter but instead pass it as query parameter. So the route will turn out to be 
```javascript
router.get('/question', auth_token, getQuestion.getQuestion);
```
So when the user is calling this API, it will be `GET /question?user_id=20` or `GET /question`